app/models/day.rb

```diff
@@ -0,0 +1,14 @@
+class Day
+
+  attr_reader :name, :date
+
+  def initialize(name, date)
+    @name = name
+    @date = date
+  end
+
+  YESTERDAY = new "yesterday", Date.yesterday
+  TODAY = new "today", Date.today
+  TOMORROW = new "tomorrow", Date.tomorrow
+  DAY_AFTER_TOMORROW = new "day_after_tomorrow", Date.tomorrow + 1
```

Comment:
What happens when the rails process runs for longer than 1 day?
Because of the way the Date instances are cached as constants they would not update...